Assets/_BForBoss/_Entities/Scripts/EnemyLifeCycleBehaviour.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using UnityEngine;
+
+namespace Perigon.Entities
+{
+    public class EnemyLifeCycleBehaviour : LifeCycleBehaviour
+    {
+        [SerializeField] private HealthbarViewBehaviour _healthbar;
+
+        //TODO - don't do this.knockback should rely on something else
+        public ILifeCycle LifeCycle => _lifeCycle;
```

Comment:
Leaving this as a reminder, there should be a Death State, on the main `FloatingTargetbehaviour` that could be used instead of this.